experimental/dds/tree2/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -460,6 +498,8 @@ function markFollowsMoves(mark: Mark<unknown>): boolean {
 		case "MoveIn":
 		case "Placeholder":
 			return false;
+		case "Transient":
+			fail("Should not rebase composite marks");
```

Comment:
Couldn't a transaction create a Transient mark, in which case we do expect them to be rebased?

Reply:
Yes, good catch. I've added a test to exercise this.